wrapper.go

```diff
@@ -32,10 +34,22 @@ const (
 func init() {
 	logger = log.New()
 	logger.Out = os.Stdout
-	logger.Formatter = &easy.Formatter{
-		TimestampFormat: "2006-01-02 15:04:05",
-		LogFormat:       "#LUMIGO# - %time% - %lvl% - %msg%\n",
+	logger.Formatter = &LogFormatter{}
+}
+
+//Log custom format
+type LogFormatter struct{}
+
+//Format details
+func (s *LogFormatter) Format(entry *log.Entry) ([]byte, error) {
+	timestamp := time.Now().Local().Format("2006-01-02 15:04:05")
```

Comment:
Are you sure that it should be `Local` and not `utc`? Can you check what do we do in the other tracers?